api/app/signals/apps/email_integrations/actions/abstract.py

```diff
@@ -51,17 +51,28 @@ def __call__(self, signal, dry_run=False):
 
         return False
 
-    def get_additional_context(self, signal, dry_run=False):
+    def mail_with_context(self, signal: Signal, dry_run=False, **kwargs):
+        """"
+        Send a system mail where additional parameters can be sent for the context
+        """
+        if dry_run:
+            return True
+
+        if self.send_mail(signal, **kwargs):
+            self.add_note(signal)
+            return True
+
+    def get_additional_context(self, signal, dry_run=False, **kwargs):
         """
         Overwrite this function if additional email context is needed.
         """
         return {}
 
-    def get_context(self, signal, dry_run=False):
+    def get_context(self, signal, dry_run=False, **kwargs):
```

Comment:
Why did you add **kwargs here? It is not needed, please revert back to the original code